src/app/customer.ts

```diff
@@ -0,0 +1,21 @@
+import { Address } from './address';
```

Comment:
Would suggest putting all interfaces in an interface folder so it won't clutter the app folder